assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -388,24 +402,26 @@ function getInnerElement(
         aria-label={alt}
         loading={'lazy'}
         onClick={onClick(thumbnailURL)}
-        onLoad={makeImageVisible}
+        onLoad={makeMediaVisible}
         {...dropTargetsBindings(thumbnailURL)}
       />
     );
   } else if (type === 'video') {
     const { lengthFormatted, poster, mimeType } = resource;
+    const displayPoster = poster ? poster : newVideoRef.current;
```

Comment:
Hm, I feel like the video is playing for a split-second before the poster is generated and the static image takes over? Or maybe the issue is, that when the poster is generated, the video stops playing even though it shouldn't?

At least the result is: When I drag a video straight from my desktop to the canvas, the video plays for a short while (a few seconds) and then stops playing. Shouldn't it either not play at all or keep playing?

Reply:
Hmm, I'm actually not seeing this -- the video doesn't have `autoplay` set and the default value is `false`. It starts playing on hovering though -- could it be that it happens when you're hovering over the uploaded video at the same time?

Reply:
I observe the same thing as @barklund (but I think it's not this PR fault, it happens also on main?)

You need a long but small in size video like [this one](https://xwp.slack.com/files/USWHNEUHH/F016TR9SN7K/timecode.mp4).
When dropped it auto-plays (like it should, right?) but it stops (resets) when the poster is generated/replaced.

Reply:
Ah, okay, now I see, you mean the dropped video on the Page plays, I thought that the video in the library started playing for some reason when you tested...

This seems to happen in main as well, yes, we can create a separate issue for that if it needs a fix.

Reply:
Alright, nothing to do with this one, then. Please file a new issue and close let's get on with this one :)